test/parallel/test-promise-hook-create-hook.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+// Flags: --expose-internals
+const common = require('../common');
+const assert = require('assert');
+const { createHook } = require('internal/promise_hook');
```

Comment:
```suggestion
const { promiseHooks } = require('async_hooks');
```
and remove `// Flags: --expose-internals`.

I think it's better to test the public API if internals are not really needed.

Reply:
Ah, yep. This came out of initially starting this as internal-only but then ended up changing halfway through. I'll fix that. :)